salt/osfingermap.yaml

```diff
@@ -12,6 +12,17 @@
 {%- set osmajorrelease = salt['grains.get']('osmajorrelease', osrelease)|string %}
 {%- set salt_repo = salt['pillar.get']('salt:repo', 'https://repo.saltproject.io') %}
 
+Debian-11:
+  pygit2: python-pygit2
+  gitfs:
+    pygit2:
+      install_from_source: true
+      version: 1.9.0
+      git:
+        require_state: false
+        install_from_package: Null
+  libgit2: libgit2-glib-1.0-0
+
```

Comment:
In case of platform updates, I think that making the new requirement the default and adding configuration for supported older ones is better.

In this case:

- the `osfamillymap.yaml` should use values working for the latest version of the mainstream OS supported by the formula (`Debian` for Debian based OS and `RedHat` for RedHat based ones)
- the `osmap.yaml` should use values working for the latest version of the OS supported by the formula (`Ubuntu-22.04` for example)
- the `osfingermap.yaml` should use values for specific older OS version supported by the formula (`Ubuntu-18.04` for example)

I remember discussing this topic somewhere with @myii or the @saltstack-formula/wg or even in an issue but can't find the link now.